scripts/tidy.sh

```diff
@@ -2,7 +2,25 @@
 
 set -euo pipefail
 
-for f in $(git ls-files | grep go.mod)
+ROOT_DIR=$(git rev-parse --show-toplevel)
+SDK_DIR="$ROOT_DIR/sdk"
+PKG_DIR="$ROOT_DIR/pkg"
+
+for f in $(git ls-files | grep 'go\.mod')
 do
-    (cd "$(dirname "${f}")" && go mod tidy -compat=1.18)
+    DIR="$(dirname "${f}")"
+    pushd "${DIR}" >/dev/null
+    case "${DIR}" in
+        pkg|sdk|developer-docs)
+            echo "tidying '${DIR}'"
+            go mod tidy -compat=1.18
+            ;;
+        *)
```

Comment:
Nice bash-foo! Could you include a line comment at the top of the loop indicating the intent of this block? I feel like I get it, but I'm not sure it would be obvious to everyone.

Reply:
Added a comment